.buildkite/commands/command-that-calls-cmd-not-installed.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = "Stop"
+
+# This should fail because npm is not available out of the box on our Windows
+# CI in 06/2025
+npm
+
+if ($LASTEXITCODE -ne 0) {
+    Write-Host "Script failed with exit code $LASTEXITCODE"
+    exit $LASTEXITCODE
+}
```

Comment:
Unless we set `$ErrorActionPreference = "Stop"`, this won't result in a failure. See [this build](https://buildkite.com/automattic/ci-toolkit-buildkite-plugin/builds/1229/steps/canvas?sid=0197a3c3-6d66-42c8-931c-b37bbad926c9)

![image](https://github.com/user-attachments/assets/96831a14-41f0-4fbf-b83a-528ff018a06f)

My guess is that's because a missing command is not considered a terminating error by default 🤷‍♂️